Objects/unicodeobject.c

```diff
@@ -683,13 +683,38 @@ backslashreplace(_PyBytesWriter *writer, char *str,
     return str;
 }
 
+static Py_ssize_t
+xmlcharrefreplace_get_incr(Py_UCS4 ch)
```

Comment:
`get_incr` isn't very descriptive, how about `get_xmlcharref_length`?

Since this is now pretty far removed from where `incr` is used, add a comment explaining it's 2+X+1 because of the `"&#%d;"` format below.